lib/mistral_ai_summary_example/mistralai_summary_page.dart

```diff
@@ -1,13 +1,165 @@
 import 'package:flutter/material.dart';
+import 'package:mistral_ai_chat_example_app/l10n/l10n.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_summary_example/summary_settings.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_summary_example/summary_settings_page.dart';
+import 'package:mistral_ai_chat_example_app/mistral_ai_summary_example/utils.dart';
+import 'package:mistralai_client_dart/mistralai_client_dart.dart';
 
-class MistralAISummaryPage extends StatelessWidget {
+class MistralAISummaryPage extends StatefulWidget {
   const MistralAISummaryPage({super.key});
 
+  @override
+  State<MistralAISummaryPage> createState() => _MistralAISummaryPageState();
+}
+
+class _MistralAISummaryPageState extends State<MistralAISummaryPage> {
+  final MistralAIClient mistralAIClient = MistralAIClient(
+    apiKey: '',
```

Comment:
```suggestion
    apiKey: const String.fromEnvironment('MISTRAL_AI_API_KEY'),
```

Reply:
Resolved in [cd824c5](https://github.com/nomtek/flutter_ai_examples/pull/3/commits/cd824c5d1c09a0b618b7653e178087c53389a0f2)